dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/dataset/Dataset.java

```diff
@@ -227,16 +227,17 @@ public void setGuestbook(Guestbook guestbook) {
     }
 
     /**
-     * Timer used mainly for OAI exporters to determinate whether metadata was changed or not,
-     *  since license in exporters could be changed based on guestbook.
-     *
+     * Timer used for OAI exporters. Holds metadata accessibility last change time.
```

Comment:
`Time of last changes that could affect exporters results, but are not related to specific dataset version.`?